location/src/main/java/org/odk/collect/location/GoogleFusedLocationClient.java

```diff
@@ -196,6 +198,7 @@ public void onConnectionSuspended(int cause) {
 
     @Override
     public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
+        Analytics.log("GoogleFusedLocationClient", "onConnectionFailed", connectionResult.toString());
```

Comment:
We'd need to add `onConnectionFailed` as a custom dimension to events in Firebase for this. I'm also a little worried about logging the `connectionResult` in case it contains a device identifier (or anything that could be PII).

Instead, could we just log a simple event like `GoogleFustLocationClientConnectionFailed`? That said, is this even something we have a question about? I'm not sure we're looking to track this at the moment.

Reply:
> I'm also a little worried about logging the connectionResult in case it contains a device identifier

> Instead, could we just log a simple event like GoogleFustLocationClientConnectionFailed? That said, is this even something we have a question about?

I wanted to know not only how often it happens but if the result contains something important to display to a user.

Reply:
but I don't have strong feelings about it so I'm ok with just checking if this takes place at all and then deciding if we need to determine the reason or just display something.